lib/content-services/document-list/components/document-list.component.ts

```diff
@@ -839,6 +846,10 @@ export class DocumentListComponent implements OnInit, OnChanges, AfterContentIni
     }
 
     canNavigateFolder(node: MinimalNodeEntity): boolean {
```

Comment:
can we refactor all this method to have only one return?

Reply:
I removed my changes from this method and refactored the performNavigation method instead. Please review. Thank you, Eugenio.